src/pages/transfer/transfer.html

```diff
@@ -2,6 +2,81 @@
+                <ion-card>
+                    <h1>1. Enter address</h1>
+                    <ion-card-header>
+                        <div>
+                            <p class="inputInfo greenBorder">FROM</p>
+                            <p class="fromAddress biggerText">{{ fromWallet.publicKey }}</p>
+                        </div>
+                        <div>
+                            <p class="inputInfo" [ngClass]="{'neutralBorder': toPublicKey === undefined, 'greenBorder': toPublicKey !== undefined && toPublicKey.length > 0}">TO</p>
+                            <ion-input class="toAddress biggerText padding" type="text" placeholder="enter address (publickey)" (ngModelChange)="resetTransferState()" [(ngModel)]="toPublicKey"></ion-input> 
+                        </div>                       
+                    </ion-card-header>
+                </ion-card>
+                <ion-card>
+                    <h1>2. Select currency</h1>
+                    <ion-card-header>   
+                        <ion-item>
+                            <ion-select class="choose-currency" [(ngModel)]="chosenCurrency" (ionChange)="setChosenCurrencyAmount()" interface="action-sheet">
+                                <ion-option *ngFor="let balance of balances" [value]="balance.currency">{{ balance.currency }}</ion-option>
+                            </ion-select> 
+                        </ion-item>     
+                    </ion-card-header>
+                </ion-card>
+                <ion-card>
+                    <h1>3. Enter amount</h1>
+                    <ion-card-header>   
+                        <p class="inputInfo" [ngClass]="{'greenBorder': enoughFunds, 'redBorder': !enoughFunds, 'neutralBorder': enoughFunds === undefined }">{{ chosenCurrency }} / {{ chosenCurrencyAmount  }}</p>
+                        <ion-input class="biggerText padding" type="number" [(ngModel)]="amount" (ngModelChange)="onAmountChanged()" placeholder="enter amount"></ion-input>  
+                    </ion-card-header>
+                </ion-card>
+                <ion-card>
+                    <h1>4. Enter password to sign</h1>
+                    <ion-card-header>   
+                        <p class="inputInfo" [ngClass]="{'neutralBorder': password === undefined, 'greenBorder': password !== undefined && password.length > 0}">INPUT HIDDEN</p>
+                        <ion-input class="biggerText padding" type="password" (ngModelChange)="resetTransferState()" [(ngModel)]="password" placeholder="enter password"></ion-input>  
+                    </ion-card-header>
+                </ion-card>
+                <ion-card>
+                    <ion-card-header> 
+                        <div class="main-loader-div" *ngIf="successMessage">
+                            <p class="transfer-success-message">{{ successMessage }}</p>
+                            <div class="loader">
+                                <div></div>
```

Comment:
Is this really the way to make the loader work :O ?

Reply:
Yes, the scss takes care of the rest. A grid with spacing is being used together with an animation delay on the nth element divided